fluss-server/src/main/java/com/alibaba/fluss/server/utils/ServerRpcMessageUtils.java

```diff
@@ -1182,6 +1182,16 @@ public static PbPartitionSpec makePbPartitionSpec(ResolvedPartitionSpec spec) {
         return pbPartitionSpec;
     }
 
+    public static ResolvedPartitionSpec toResolvedPartitionSpec(PbPartitionSpec pbPartitionSpec) {
```

Comment:
As `ClientRpcMessageUtils`  also contains same method, I suggested move this method to `CommonRpcMessageUtils` to reuse.